package.json

```diff
@@ -31,6 +31,7 @@
     "@babel/core": "^7.9.0",
     "@babel/preset-env": "^7.9.5",
     "@babel/preset-typescript": "^7.9.0",
+    "@happy-dom/jest-environment": "^8.1.0",
```

Comment:
for https://github.com/nextcloud/nextcloud-browser-storage/pull/200 I used jsdom but looks like this package is a viable alternative :+1:

Reply:
Yes, both are good packages for jest environment, but happydom is much faster (some corner cases of the DOM API are not supported but we are not using them so we can enjoy the performance benefits :) )

Reply:
Awesome!